src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java

```diff
@@ -0,0 +1,153 @@
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion.NavigationBar;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.minieditor.MiniEditorComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.SignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.messagewall.MessageWall;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization.SpecialAnonymizationUserPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.ConfirmationModalPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.HelpPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.SpecialRenameUserPage;
+
+import jdk.nashorn.internal.runtime.regexp.RegExp;
+import org.assertj.core.util.Compatibility;
+import org.joda.time.DateTime;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testng.annotations.Test;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.UUID;
+
+@SuppressWarnings("SpellCheckingInspection")
+@InBrowser(browser = Browser.CHROME)
+@Execute(onWikia = "communitytest")
+@Test(groups = "anonymization")
+public class AnonymizationTests extends NewTestTemplate {
+
+  private SpecialAnonymizationUserPage openAnonymizationPage(User user) {
+    WikiBasePageObject base = new WikiBasePageObject();
+    base.loginAs(user);
+    return new SpecialAnonymizationUserPage().open();
+  }
+
+  private static final String ERROR_MESSAGE =
+      "We don't recognize these credentials. Try again or register a new account.";
+
+  @Test
+  public void anonymizedUserCannotLogin() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    SpecialAnonymizationUserPage anonymizationStaff = openAnonymizationPage(User.SUS_STAFF);
+    anonymizationStaff.fillFutureAnon(qanon)
+        .submitAnonymization();
+
+    Assertion.assertStringContains(anonymizationStaff.getAnonConfirmation(), qanon);
+
+    anonymizationStaff.logOut();
+
+    SignInPage signIn = openLoginModalOnDesktop();
+    signIn.login(qanon, passw);
+    Assertion.assertEquals(signIn.getError(), ERROR_MESSAGE);
+  }
+
+  private SignInPage openLoginModalOnDesktop() {
```

Comment:
either move to page object or use a flow below directly: `new DetachedSignInPage(new NavigationBar().clickOnSignIn());`